src/plugins/interaction/GraphInteraction/GraphInteraction.cpp

```diff
@@ -42,19 +43,29 @@
 #include <scrimmage/proto/ProtoConversions.h>
 #include <scrimmage/msgs/Graph.pb.h>
 
-
+#include <typeinfo>
 #include <memory>
 #include <limits>
 #include <iostream>
 #include <fstream>
+#include <deque>
```

Comment:
is this used anywhere?

Reply:
Good call. Removed in upcoming push. Thanks!